app/views/tag/_tags.html.erb

```diff
@@ -1,7 +1,7 @@
 <ul class= "list-inline">
   <% tags.each do |tag| %>
     <% if tag.class == NodeTag %>
-
+    <!-- tag list in sidebar -->
```

Comment:
I think this file is not part of this PR

Reply:
Is this part of the PR? I think it is not related.

Reply:
```suggestion

```

Reply:
Agreed i think it was a mistake.

Reply:
@IshaGupta18 please remove this line then this file will not be shown on github.